### PR TITLE
Take title from ascidoc first level title

### LINKED ISSUE #2: Take title from ascidooc first level title
Currently, the Asciidoc title is not adopted as the document title.

### PR DESCRIPTION
**Pull request by ursjoss**
resolves #2

paired with @fliiiix

### CODE REVIEW

**Comment by ursjoss** [comment]
Note: when downloading files (#5), titles are not yet extracted correctly. That should be covered in #5 though.

**Comment by fliiiix** [comment]
Looks like a clean Solution!

File: lib/models/note.js [hunk 1]
```diff
@@ -230,15 +231,15 @@ module.exports = function (sequelize, DataTypes) {
       },
       parseNoteInfo: function (body) {
         var parsed = Note.extractMeta(body)
-        var $ = cheerio.load(md.render(parsed.markdown))
+        var $ = cheerio.load(adoc.load(parsed.markdown,{ attributes: {showtitle: true}}).convert())
```
Comment (fliiiix) [inline lib/models/note.js]: What is the difference between `adoc.load().convert()` and `adoc.convert()` ?

**Comment by ursjoss** [comment]
@fliiiix excellent point: `adoc.load().convert()` does the same thing as `adoc.convert()`. The longer version was based on a series of refactorings I have not completly simplified yet. The third commit does that now... Thanks!

**Comment by madmas** [comment]
sweet thing! thanks a lot! :)

**Pull request merged**